.changes/unreleased/bridge-TINY-11847-2025-03-21.yaml

```
@@ -0,0 +1,6 @@
+project: bridge
+kind: Added
+body: Now `BaseMenuButtonInstanceApi` as `setTooltip`.
```

I feel like more than just this one thing changed, and also that some of the changes belong in the tinymce changelog 🤔 

```suggestion
body: Menu buttons now have a `setTooltip` API.
```

---

changed